.github/workflows/release-python.yml

```diff
@@ -0,0 +1,78 @@
+name: Python Wheels
+
+on: [push, pull_request]
+
+jobs:
+  build-wheels:
+    runs-on: ${{ matrix.os }}
+    strategy:
+      matrix:
+        os: [macos-latest, windows-2019, ubuntu-latest]
+    name: Build CPython ${{ matrix.python-version }}-${{ matrix.os }}
+    steps:
+      - uses: actions/checkout@v2
+      - uses: pypa/cibuildwheel@v1.12.0
+        env:
+          CIBW_ARCHS_MACOS: x86_64 universal2
+          CIBW_TEST_SKIP: '*universal2:arm64'
+          CIBW_BUILD: "cp36-macosx_x86_64 cp38-macosx_universal2 cp36-win* cp36-manylinux_{x86_64,i686}"
+      - uses: actions/upload-artifact@v2
+        with:
+          name: python_bsonjs-3.6-${{ matrix.os }}-wheel
+          path: ./wheelhouse/*.whl
+          if-no-files-found: error
+  test-non-linux-wheels:
```

Comment:
But then that would only test our wheels on the configurations that we are building on. Because we are building abi3 wheels we want to ensure that they work on multiple different versions of python. I'm not sure how to both make it only make wheels with python 3.6 (`cp36*`) but also to test that same wheel on python 3.7 for example. There seems to be a 1-1 correspondence between wheels and testing and environments.

Reply:
Oh great point. Testing cross combinations is important too.